types/chrome/index.d.ts

```diff
@@ -2074,11 +2087,11 @@ declare namespace chrome.declarativeContent {
 declare namespace chrome.declarativeWebRequest {
     export interface HeaderFilter {
         nameEquals?: string | undefined;
-        valueContains?: any;
+        valueContains?: string | string[];
```

Comment:
https://developer.chrome.com/docs/extensions/reference/declarativeWebRequest/#type-HeaderFilter

Reply:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        valueContains?: string | string[] | undefined;
```